modules/developer_manual/pages/mobile_development/ios_library/library_installation.adoc

```diff
@@ -45,18 +45,21 @@ The following files are required:
 * `readFolder` and `readFile` methods
 * `OCFrameworkConstants.h` (Customize constants)
 
-2. Add the library file to the project. From the `Build Phases` tab, scroll to `Link binary files` and select the `+` to add a library.
+[start=2]
```

Comment:
what is that kind of directive used [start=2] ?

Reply:
That formatting can be used to ensure that [ordered lists are generated consecutively](https://asciidoctor.org/docs/user-manual/#ordered-lists) when they're not written one after another.